### PR TITLE
Secure removal of `type` is `prerender` events

### PR DESCRIPTION
**Pull request by PEM--**
This fixes the issue #91.

### CODE REVIEW

**Comment by dmvaldman** [comment]
Ah, is this to prevent removing a type that was never added? In which case, nice check! We'd like to add it. Not sure what the `prerender` event has to do with it though.

Unfortunately it doesn't pass our Linter as you can see in the Travis warning. It's just the space in front of the `if` in

```
if(listener)
```

We could fix here, but I'd rather give you the opportunity to change on your side.

**Comment by PEM--** [comment]
@dmvaldman, it looks better now. Travis status is fine. Sorry for the missed linting rule. I didn't notice it.

Concerning the `type` is `prerender`, it refers to the issue Famous/famous#91 in the parent git repo (Famous/Famous).

Here is a simple example to reproduce the issue:

```
var foo = Timer.debounce(function() {
    console.log('i should only show up once')
}, 1000);
foo()
foo()
foo()
```

File: EventEmitter.js [hunk 1]
```diff
@@ -72,8 +72,11 @@ define(function(require, exports, module) {
      * @return {EventEmitter} this
      */
     EventEmitter.prototype.removeListener = function removeListener(type, handler) {
```
Comment (PEM--) [inline EventEmitter.js:74]: `type` value was `prerender` and it was not available in the `EventEmitter` instance.

**Comment by PEM--** [comment]
@dmvaldman Is it OK for you?

**Comment by dmvaldman** [comment]
Approved @TheAlphaNerd

**Comment by MylesBorins** [comment]
Merged into Internal, expect to see this publicly very very very soon 
:+1:

**Comment by PEM--** [comment]
Thanks guys, you rock :rocket:

**Pull request closed without merging**